packages/contractkit/src/identity/offchain/schemas/binary.ts

```diff
@@ -0,0 +1,58 @@
+import { Err, makeAsyncThrowable, Ok } from '@celo/base/lib/result'
+import OffchainDataWrapper from '../../offchain-data-wrapper'
+import { OffchainError } from './errors'
+import {
+  buildEIP712TypedData,
+  EncryptedSchema,
+  readEncrypted,
+  Schema,
+  signBuffer,
+  writeEncrypted,
+} from './utils'
+
+/**
+ * Schema for writing any generic binary data
+ */
+export class BinarySchema implements Schema<Buffer> {
```

Comment:
Related to my other comment, we might want to call this `UnencryptedBinarySchema`.